# Provided by the Traffic Manager, this info provides updates whenever there
# is a new traffic level for one of the lanelets

int32 lanelet
int32 new_traffic_level